keyboards/cest73/tkm/config.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xBADD
+#define PRODUCT_ID      0xAA55
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    cest73
+#define PRODUCT         "Ten Key Mirrored 2021 Release 1.03"
+
+/* key matrix size */
+#define MATRIX_ROWS 11
+#define MATRIX_COLS 10
+
+/* matrix sz handy ruler:  1   2   3   4   5   6   7   8   9  10  11  12 */
+/* key matrix pins */
+/*    row    handy ruler: r0  r1  r2  r3  r4  r5  r6  r7  r8  r9  r10 */
+#define MATRIX_ROW_PINS { B0, B1, B2, B3, B4, B5, B6, B7, C6, C7, D0 }
+/*    column handy ruler: c0  c1  c2  c3  c4  c5  c6  c7  c8  c9 */
+#define MATRIX_COL_PINS { D1, D2, D3, D4, D5, D6, D7, F0, F1, F4 }
+#define UNUSED_PINS { AF }
+
+/* the LEDs:
+ *  PF5 - CAPS
+ *  PF6 - NUM
+ *  PF7 - SCLK
+ *  PE6 - Backlight */
+
+#define BACKLIGHT_PIN E6
```

Comment:
Note that `E6` is not the best choice for the backlight pin on the ATmega32U4 chip. Ideally you should choose a pin which can be connected to one of PWM outputs of timers 1 or 3 — i.e., one of `B5`, `B6`, `B7`, `C6`; then the backlight code would use the corresponding timer to generate the PWM signal in hardware. Using any other pin would require the software PWM emulation (the code enables timer interrupts and toggles the specified pin from interrupt handlers), which mostly works, but can result in visible flickering of backlight if you have some code which disables interrupts for a significant time (but the main offender is the WS2812 driver, which you don't have here, so it might still be acceptable).

You may consider changing the backlight pin if you ever get to making a rev2 of the PCB.

Reply:
Thanks for the advice, will keep in mind if i ever go for an updated PCB, otherwise works well, typing this on the very board.